### DISCUSSION TITLE
Is it possible to run on Cloudflare Workers?

### DISCUSSION THREAD

**Discussion by BrandonStudio** [searxng/searxng#4119] [Q&A] [answered]
Is it possible to do some adaptation to make this repo able to deploy to Cloudflare Workers?

**Comment by return42** [comment]
Not sure how often we have repeated .. SearXNG was made for privacy ..

### Don't use Cloudflare! 

### Cloudflare can read everything (because they MITM your https connection) .. which contrasts with any efforts to preserve privacy.

further Cloudflare is a source of problems /  certainly also because most users are only superficially familiar with it and are inexperienced in its use. : 

- https://github.com/searxng/searxng/issues/3730
- https://github.com/searxng/searxng/issues/1113#issuecomment-1101357319
- https://github.com/searxng/searxng/issues/1680#issuecomment-1220637492
- https://github.com/searxng/searx-instances/issues/85#issuecomment-1154107294

**Comment by unixfox** [comment, marked as answer]
No you can't because cloudflare workers are short lived environments. And searxng needs to continuously run for it to work.

**Reply by BrandonStudio** [reply]
Apparently I had misunderstood the functionality of this repo. Does it contain a full frontend web interface?

For me I just want to aggregate search result for individual requests, so maybe that is possible? And for this I just need a small portion of the code?

**Reply by unixfox** [reply]
Yes it's a full frontend web interface and you can't run it without it.
